jobs/bodhi-trigger.Jenkinsfile

```diff
@@ -189,7 +189,8 @@ cosaPod(cpu: "0.1", kvm: false) {
             """)
         }
         if (test.result != 'SUCCESS') {
-            pipeutils.matrixSend(":fire: [:ocean:](${blueocean_url}) ${currentBuild.description} - ${outcome} ")
+            def bodhi_url="https://bodhi.fedoraproject.org/updates/${msg.update.updateid}"
+            pipeutils.matrixSend("🔥 ${currentBuild.description} - ${outcome}  🌊 ${blueocean_url} 🪷 ${bodhi_url}")
```

Comment:
```suggestion
            pipeutils.matrixSend("🔥 ${currentBuild.description} - ${outcome}  [🌊](${blueocean_url}) [🪷](${bodhi_url}"))
```

Reply:
That won't work because maubot does not apply formatting 

https://github.com/coreos/coreos-ci/issues/67#issuecomment-2164478081

Reply:
Ohh yes, the format is set up at the plugin level, not part of the request.